code/modules/Economy/Accounts.dm

```diff
@@ -383,6 +384,14 @@ var/global/datum/money_account/trader_account
 			if("toggle_account")
 				if(detailed_account_view)
 					detailed_account_view.disabled = detailed_account_view.disabled ? 0 : 2
+			if("edit_wage_payout")
+				var/acc_num = text2num(href_list["account_num"])
+				var/datum/money_account/acc = get_money_account_global(acc_num)
+				if(acc)
+					var/new_payout = input(usr, "Select a new payout for this account", "New payout", acc.wage_gain) as null|num
```

Comment:
please round this, because I do not think our credit system can handle fractional credits